eva/readers/opencv_reader.py

```diff
@@ -59,7 +59,11 @@ def _read(self) -> Iterator[Dict]:
                 _, frame = video.read()
                 frame_id = begin
                 while frame is not None and frame_id <= end:
-                    yield {"id": frame_id, "data": frame}
+                    yield {
+                        "id": frame_id,
+                        "data": frame,
+                        "seconds": frame_id // video.get(cv2.CAP_PROP_FPS),
```

Comment:
Are we planning to add a `seconds` column? If yes, we need to register it in the catalog, or the rest of the system won't be aware of it. Add it https://github.com/georgia-tech-db/eva/blob/04368ef72a04973b5ad3bfdf98b34e957440539e/eva/catalog/catalog_utils.py#L27

Btw we should not hardcode these strings in the reader, but rather get them from the catalog object. Something to do in the future. I'll take note of it.

Reply:
Yes, this is done.